mlflow/store/artifact/dbfs_artifact_repo.py

```diff
@@ -23,12 +24,14 @@
 )
 import mlflow.utils.databricks_utils
 
+# The following constants are defined as @developer_stable
 LIST_API_ENDPOINT = "/api/2.0/dbfs/list"
 GET_STATUS_ENDPOINT = "/api/2.0/dbfs/get-status"
 DOWNLOAD_CHUNK_SIZE = 1024
 USE_FUSE_ENV_VAR = "MLFLOW_ENABLE_DBFS_FUSE_ARTIFACT_REPO"
```

Comment:
We might want to add `constants.py`. Adding `The following constants are defined as @developer_stable` to every single constant is exhausting.

Reply:
We're planning on adding the constants module (probably next sprint?) but we'll have to leave all of these for at least 3-4 releases so as to not break existing integrations that directly import these. We discussed the details of this with the AML folks :)